static/app/components/core/slider/index.chonk.tsx

```diff
@@ -0,0 +1,294 @@
+import {
+  type CSSProperties,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import {mergeRefs} from '@react-aria/utils';
+
+import {space} from 'sentry/styles/space';
+import {chonkStyled} from 'sentry/utils/theme/theme.chonk';
+
+import type {SliderProps} from './index';
+
+const passthrough = (n: number | '') => n;
+
+export function Slider({
+  formatLabel = passthrough,
+  ref,
+  ...props
+}: SliderProps & {ref?: React.Ref<HTMLInputElement>}) {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const refs = mergeRefs(ref, inputRef);
+  const step = toNumber(props.step ?? -1);
+  const {value, min, max} = resolveMinMaxValue(props);
+  const [label, setLabel] = useState(value);
+  const initialProgress = getProgress(value, min, max);
+  const filledSteps = useMemo(() => (step === -1 ? -1 : label / step), [step, label]);
+
+  const handleChange = useCallback(
+    (event: Event) => {
+      const input = event.target as HTMLInputElement;
+      const {valueAsNumber, min: nativeMin, max: nativeMax} = input;
+      setLabel(valueAsNumber);
+      const progress = getProgress(valueAsNumber, nativeMin || 0, nativeMax || 100);
+      input.parentElement?.style.setProperty('--p', `${progress.toFixed(0)}%`);
+    },
+    [setLabel]
```

Comment:
`setLabel` isn’t needed because it’s a `useState` setter, so the linter knows it’s stable.

```suggestion
    []
```